docs/build-mvm/tutorials/event-handling-in-js/1-event-handling.md

```diff
@@ -0,0 +1,112 @@
+# Introduction
+
+> Make sure you've read ***Getting Started*** and ***Quickstart*** pages.
+
+Fluence Cloud Functions are hosted on distributed peers in the Fluence network waiting to be called. That is, your functions need to be triggered in order to do some work. Event triggers may arise from a multitude of sources ranging from browser click-events to changes in a database column. Regardless of the source, the event needs to trigger the function(s) which, as you already know from the introduction and quickstart chapters, requires Aqua to orchestrate the invocation of your distributed compute function(s).
+
+One way of triggering the orchestration of your function(s) is interactively with the Fluence CLI: `fluence run -f 'your function(args)'`. However, events may require (near) real-time processing which in turn requires the orchestration of your function(s) immediately after the event occurred. Whether this is in the browser or some other application, we need to trigger the appropriate Aqua script from a (embedded) client peer.
```

Comment:
```suggestion
One way of triggering the orchestration of your functions is interactively with the Fluence CLI: `fluence run -f 'your function(args)'`. However, events may require near real-time processing which in turn requires the orchestration of your functions immediately after the event occurred. Whether this is in the browser or some other application, we need to trigger the appropriate Aqua script from a embedded client peer.
```

Reply:
I'd recommend not to use the excessive braces like that. There's no semantic advantages in hinting at alternatives (like singular/plural), or may even mislead, but extra syntactic elements get in the way when reading.

Reply:
Need to ask @boneyard93501 is there any specific meaning in these braces.